src/contracts/contract_deployer.star

```diff
@@ -45,16 +96,11 @@ def launch_contract_deployer(
                 "sleep 3",
                 "web3 transfer $FUND_VALUE to $GS_PROPOSER_ADDRESS",  # Fund Proposer
```

Comment:
wondering if it's possible to fund the addresses once in the factory contract deployment step instead of each time so this step is faster? dont know how long the transfer step is tho

Reply:
it’s not this web3 call that takes long, but the factory deployed script which can only be deployed on a finalized chain.

Reply:
makes sense, prob not worth doing then

Reply:
wdyt about refactoring to perform the "private key" retrieval `run_sh`'s once:
```
    gs_sequencer_private_key = plan.run_sh(
        description="Getting the sequencer private key",
        run="cat /network-configs/GS_SEQUENCER_PRIVATE_KEY ",
        files={"/network-configs": op_genesis.files_artifacts[0]},
    )
```
private key stays the same cause of `ADMIN`, `BATCHER`, and `PROPOSER` are same for all chains no? then we dont have to spin up/teardown the unecessary task containers?

Reply:
Private key lookup can be done just once indeed.